src/containers/ContentLoader.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import ContentLoader from "react-content-loader";
+
+export function ChartLoader({
```

Comment:
@petrKavulok How this will look on small / medium / large cards and screens?

Reply:
https://user-images.githubusercontent.com/79644454/154438699-31b38524-c32d-4b50-858a-08de35bc7c32.mov